### PR TITLE
Remove airplane banner

### PR DESCRIPTION
**Pull request by megcrenshaw**
Removes the airplane banner from our repo.
image

This PR https://github.com/code-dot-org/code-dot-org/pull/44847 updated the airplane banner to the superhero banner, but the elementary school page still had the airplane banner (presumably because PL has a different process for elementary teachers). Here, I delete it from the elementary school page and completely remove the banner from our repo.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by TurnerRiley — APPROVED** [review]

File: apps/src/templates/ProfessionalLearningApplyBanner.jsx [hunk 1]
```diff
@@ -1,160 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import './professionalLearningApplyBanner.scss';
```
Comment (davidsbailey) [inline apps/src/templates/ProfessionalLearningApplyBanner.jsx:3]: also remove .scss file?

**Review by davidsbailey — APPROVED** [review]

File: apps/src/templates/ProfessionalLearningApplyBanner.jsx (hunk 1 above)
Comment (megcrenshaw) [reply apps/src/templates/ProfessionalLearningApplyBanner.jsx:3]: Ah yes, good call!

**Review by kelbyhawn — APPROVED** [review]
Looks good to me, but what prompted this removal? (Asking in case someone asks me why it was removed 😄)

**Comment by megcrenshaw** [comment]
> Looks good to me, but what prompted this removal? (Asking in case someone asks me why it was removed 😄)

Good question! I believe @dmcavoy has more info, but I found this "We are going to hold on banners until the 1/10 national launch. We will be using the Superhero banner so we will want to make sure we think through which features from the old banner we want to move over. I'll add a task for me to think through this and create tasks for this update" from Dani –– not sure how that relates to the airplane banner

**Comment by megcrenshaw** [comment]
I ended up removing the plane as well, as it's now not being used. @kelbyhawn Does that sound okay?

**Comment by dmcavoy** [comment]
The airplane banner was the old banner which we moved away from in favor of the new superhero banners. The airplane banner was still showing on at least one page which reminded me we should remove it.

**Comment by kelbyhawn** [comment]
Thanks for the extra context @megcrenshaw and @dmcavoy! 🙌

**Pull request merged**